sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Framework/Validators/EnumberableItemsNotNullAttribute.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Linq;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.Framework.Validators
+{
+    /// <summary>Validator to ensure that a type value is enumerable and contains no null items.</summary>
+    [AttributeUsage(AttributeTargets.Property)]
+    internal class EnumberableItemsNotNullAttribute : ValidationAttribute
```

Comment:
```suggestion
    internal class EnumerableItemsNotNullAttribute : ValidationAttribute
```

Reply:
Updated in [PR](https://github.com/hakimms/azure-sdk-for-net/pull/2)